bin/dev

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH.unshift(File.expand_path("../../lib", __FILE__))
```

Comment:
Using `require "bundler/setup"` would be better I think because it takes care of including gems mentioned in Gemfile into `LOAD_PATH` a well.